Justfile

```diff
@@ -74,9 +74,9 @@ all-release: build-release test-release
 # build the man pages
 @man:
     mkdir -p "${CARGO_TARGET_DIR:-target}/man"
-    version=$(cat Cargo.toml | grep ^version | head -n 1 | awk '{print $NF}' | tr -d '"'); \
+    version=$(awk 'BEGIN { FS = "\"" } ; /^version/ { print $2 ; exit }' Cargo.toml); \
     for page in eza.1 eza_colors.5 eza_colors-explanation.5; do \
-        pandoc --standalone -f markdown -t man <(cat "man/${page}.md" | sed "s/\$version/v${version}/g") > "${CARGO_TARGET_DIR:-target}/man/${page}"; \
+        sed "s/\$version/v${version}/g" "man/${page}.md" | pandoc --standalone -f markdown -t man > "${CARGO_TARGET_DIR:-target}/man/${page}"; \
```

Comment:
That might be an improvement ... just wondering though, is there any other reason for this than older bash versions not having process substitution?

Reply:
That's a good practice :)
https://github.com/koalaman/shellcheck/wiki/SC2002

Reply:
And i forgot the most important…
Yes it works in bash, but not with `just` (at least for me…). Even a basic Justfile with the "man" fails :
```
cat Justfile
# build the man pages
@man:
    mkdir -p "${CARGO_TARGET_DIR:-target}/man"
    version=$(cat Cargo.toml | grep ^version | head -n 1 | awk '{print $NF}' | tr -d '"'); \
    for page in eza.1 eza_colors.5 eza_colors-explanation.5; do \
        pandoc --standalone -f markdown -t man <(cat "man/${page}.md" | sed "s/\$version/v${version}/g") > "${CARGO_TARGET_DIR:-target}/man/${page}"; \
    done;
```
```
just man
sh: 1: Syntax error: "(" unexpected
error: Recipe `man` failed on line 7 with exit code 2
```

just --version **1.14.0**

Reply:
I also use just v1.14.0 so that can't be the problem. ... As I said in the related issue my guess would be the bash version ... which one do you use?

Reply:
ah just saw your comment in the issue ... interesting

Reply:
Yeah, also sorry for the cross-posts…

Bash version : **5.2.15**.

And for the test, i put "build the man pages" commands to a tiny script shell : 
``` sh
cat pandoc.sh
#!/bin/bash -e

# build the man pages
rm -rf "${CARGO_TARGET_DIR:-target}/man"
mkdir -p "${CARGO_TARGET_DIR:-target}/man"
version=$(cat Cargo.toml | grep ^version | head -n 1 | awk '{print $NF}' | tr -d '"'); \
        for page in eza.1 eza_colors.5 eza_colors-explanation.5; do \
                pandoc --standalone -f markdown -t man <(cat "man/${page}.md" | sed "s/\$version/v${version}/g") > "${CARGO_TARGET_DIR:-target}/man/${page}"; \
done;
```

And… it works :sweat_smile: 
``` sh
./pandoc.sh

ls -l target/man/
total 28
-rw-r--r-- 1 root root 11486 Oct  2 14:45 eza.1
-rw-r--r-- 1 root root  2893 Oct  2 14:45 eza_colors-explanation.5
-rw-r--r-- 1 root root  8775 Oct  2 14:45 eza_colors.5
```